lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -34,7 +35,9 @@ class PartialFinder < TemplateFinder
         # @since 0.4.3
         # @api private
         def find
-          if path = partial_template_under_view_path
+          if cached_template = find_cached_template
+            cached_template
+          elsif path = partial_template_under_view_path
```

Comment:
@stevehook As consequence of [my comment above](https://github.com/lotus/view/pull/86/files#r48692258), this `elsif` logic should be removed.

Please ensure that `PartialTemplatesFinder` implements the logic of `partial_template_under_view_path`.